src/pyk/ktool/kompile.py

```diff
@@ -254,27 +254,27 @@ class LLVMKompileType(Enum):
 class LLVMKompile(Kompile):
     base_args: KompileArgs
     llvm_kompile_type: LLVMKompileType
-    llvm_kompile_output: str | None
+    llvm_kompile_output: Path | None
     opt_level: int
     ccopts: tuple[str, ...]
     no_llvm_kompile: bool
     enable_search: bool
     enable_llvm_debug: bool
+    llvm_proof_hint_instrumentation: bool
 
     def __init__(
         self,
         base_args: KompileArgs,
         *,
-        llvm_kompile_type: str | LLVMKompileType | None = None,
-        llvm_kompile_output: str | None = None,
+        llvm_kompile_type: LLVMKompileType | None = None,
+        llvm_kompile_output: Path | None = None,
```

Comment:
```suggestion
        llvm_kompile_type: str | LLVMKompileType | None = None,
        llvm_kompile_output: str | Path | None = None,
```

I like having these strings, it makes scripting / interpreter use very convenient because then calling `kompile` does not require the ceremony of importing everything.

Reply:
I agree, for production code, we should definitely write `llvm_kompile_output=LLVMKompileType.C`, but it is still very convenient from time to time to just drop in an interpreter and write

```
> kompile(llvm_kompile_type='c')
```

and not go through all the ceremony.

For `llvm_kompile_output`, `Path` does not have any advantage in type checking against `str | Path`.

Reply:
Done.